hoomd/md/methods.py

```diff
@@ -77,6 +83,14 @@ class NVT(_Method):
 
         tau (float): Coupling constant for the Nosé-Hoover thermostat. (in time
             units).
+
+        translational_thermostat_dof (tuple[float, float]): Additional degrees
+            of freedom for the translational thermostat (:math:`\xi`,
+            :math:`\eta`)
+
+        rotational_thermostat_dof (tuple[float, float]): Additional degrees
+            of freedom for the rotational thermostat (:math:`\xi_\mathrm{rot}`,
+            :math:`\eta_\mathrm{rot}`)
```

Comment:
We may want to include a sentence in the docstring explaining that these variables control the internal state of the thermostat. Not sure how much detail is appropriate, but at least mentioning that they're explained in detail in the paper.

Reply:
Aha these are well documented in the thermalize documentation. So maybe just add a note 
```suggestion
            :math:`\eta_\mathrm{rot}`) (see `~.thermalize_extra_dof` for details).
```
Also add to the translational DOF.

Reply:
I moved the longer description up into the main class documentation so it appears first and references the attribute doc.